testcases/kdump/lib/setup.sh

```diff
@@ -82,6 +82,11 @@ fi
 
 make
 
+
+if [ -f /etc/issue ]; then
+    dvar=`cut -d' ' -f1 /etc/issue | head -1`
+fi
+
```

Comment:
```/etc/issue``` is not a reasonable file to figure out what distribution you are running on.

Modern enough distributions should have ```/etc/os-release``` have a look at:
https://www.freedesktop.org/software/systemd/man/os-release.html